server/sendReview.js

```diff
@@ -0,0 +1,20 @@
+import express from "express";
+import fetch from "node-fetch";
+
+export const sendReviewServer = express.Router();
+
+sendReviewServer.post("/movies/:id/review", async (req, res) => {
+  await fetch("https://plankton-app-xhkom.ondigitalocean.app/api/reviews", {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify(req.body),
+  });
+});
```

Comment:
This route handler never returns a response, because it never interacts with the `res` object passed to it from express. This means that this will wait forever, until the browser or server times out.

An HTTP server should always respond to requests, even if that response contains no data. That is a core part of the contract between client and server, which is not being honored here. It's an easy mistake to make, and it will usually cause errors in the browser and/or the server, so look out for those error messages!